prometheus/generic/setup-haproxy-exporter/tasks/docker.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Run the haproxy_exporter
+  docker_container:
+    name: haproxy_exporter
+    image: "{{ haproxy_exporter_image }}:{{ haproxy_exporter_image_version }}"
+    pid_mode: "host"
```

Comment:
Is this required? Do we need to share the host's process namespace?

Reply:
Removed this  pid_mode: from task